nixos/modules/services/web-servers/apache-httpd/default.nix

```diff
@@ -318,7 +318,12 @@ let
   '';
 
 
-  confFile = pkgs.writeText "httpd.conf" ''
+  confFile = pkgs.writeTextFile {
+    name = "httpd.conf";
+    checkPhase = ''
+      ${httpd}/bin/apachectl configtext $out
```

Comment:
`${httpd}/bin/apachectl configtest -f $out`

Reply:
For future reference in case anyone sees this I'd like to correct my mistake I noticed. If specifying the config file you have to use `${httpd}/bin/httpd -t -f $out` instead of the apachectl wrapper (which assumes a hard coded location for the config file).